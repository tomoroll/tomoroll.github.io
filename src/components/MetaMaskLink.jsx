import React from 'react';

const MetaMaskLink = () => (
  <a
    href="https://metamask.io/"
    target="_blank"
    rel="noopener noreferrer"
  >
    Meta Mask
  </a>
);

export default MetaMaskLink;
