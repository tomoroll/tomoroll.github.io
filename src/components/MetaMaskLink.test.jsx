import React from 'react';
import renderer from 'react-test-renderer';
import MetaMaskLink from './MetaMaskLink';

it('renders correctly', () => {
  const tree = renderer
    .create(<MetaMaskLink />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
