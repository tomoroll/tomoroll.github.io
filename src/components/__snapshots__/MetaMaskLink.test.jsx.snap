// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<a
  href="https://metamask.io/"
  rel="noopener noreferrer"
  target="_blank"
>
  Meta Mask
</a>
`;
